rules/hoc-transliteration.js

```diff
@@ -0,0 +1,102 @@
+( function ( $ ) {
+	'use strict';
+
+	var hoctransliteration = {
+		id: 'hoc-transliteration',
+		name: 'Ho Transliteration',
+		description: 'Ho transliteration input',
+		date: '2016-10-23',
+		URL: 'http://github.com/wikimedia/jquery.ime',
+		author: 'Mangu Purty',
+		license: 'GPLv3',
+		version: '1.0',
+		contextLength: 2,
+		maxKeyLength: 3,
+		patterns: [
+			[ 'a', '𑣁'],
+			[ 'b', '𑣗' ],
+			[ 'c', '𑣏' ],
+			[ 'd', '𑣑' ],
+			[ 'e', '𑣈' ],
+			[ '𑣒h', 't',  '𑣕' ], //for ot
+			[ 'g', '𑣋' ],
+			[ 'h', '𑣙' ],
+			[ 'i', '𑣆' ],
+			[ '([𑢪-𑢿]|[𑣊-𑣟])i', '$1𑣂' ], // for wi
```

Comment:
Since I could'nt run the test myself, I don't know if there are any errors in the code, please inform me if there are any.